tests/shell/shell_recorder/README.md

```diff
@@ -0,0 +1,331 @@
+# Shell Recorder #
+
+## Configuration ##
+
+### Mountpoint: ###
+
+This is the only configuration Variable that has to be set. It is used to determine where the shell_recorder should look for changes.
```

Comment:
Please add some introduction what a "configuration Variable" is in the context of a shell_recorder.